.github/workflows/rust.yml

```diff
@@ -203,3 +203,27 @@ jobs:
             echo "Rerun 'make docs'."
             exit 1
           fi
+  test-rpm-build:
+    name: "RPM build"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Set up environment
+        run: .github/workflows/env-setup
```

Comment:
Are we using any features of this script, or are we just doing it this way for consistency?  If the latter, I'd be inclined to just use `options: --privileged`; it'd make the job easier to understand.